src/sidebar/components/test/sidebar-content-error-test.js

```diff
@@ -52,7 +52,9 @@ describe('SidebarContentError', () => {
 
     const clearButton = findButtonByText(wrapper, 'Show all annotations');
     assert.isTrue(clearButton.exists());
-    assert.equal(clearButton.props().onClick, fakeStore.clearSelection);
+
+    clearButton.props().onClick();
+    assert.called(fakeStore.clearSelection);
```

Comment:
The previous assert didn't work after the other changes in this PR due to the way that the proxy's `clearSelection` is a different function (a wrapper) than the store's `clearSelection`, so I had to change it to check the same thing a different way.